### PR TITLE
Document the usage of `.env` file

### LINKED ISSUE #6: Getting started doc should mention how to use environment variables
[getting-started.md](getting-started.md) should explain the preferred way of handling env variables as implemented in https://github.com/adobe/helix-cli/issues/580

### PR DESCRIPTION
**Pull request by dplaton**
Fixes #6

### CODE REVIEW

**Comment by tripodsan** [comment]
in general, the `.env` variables are added the `process.env`. so you can use it for any environment variable.

currently, only `hlx deploy` and `hlx publish` use the automatic env mapping from `HLX_*` env variables to yargs args (see https://github.com/yargs/yargs/blob/master/docs/api.md#envprefix).

#### the relevant arguments for `hlx publish` are:

`HLX_WSK_NAMESPACE`
`HLX_WSK_AUTH`
`HLX_WSK_HOST`
`HLX_FASTLY_AUTH`
`HLX_FASTLY_NAMESPACE`

less useful:
`HLX_REMOTE`
`HLX_API_PUBLISH`

#### the relevant arguments for `hlx deploy` are:

`HLX_WSK_NAMESPACE`
`HLX_WSK_AUTH`
`HLX_WSK_HOST`
`HLX_LOGGLY_AUTH`
`HLX_LOGGLY_HOST`
`HLX_FASTLY_AUTH`
`HLX_FASTLY_NAMESPACE`
`HLX_CIRCLECI_AUTH`

less useful:
`HLX_TARGET`
`HLX_PACKAGE`

**Comment by dplaton** [comment]
@rofe Yes, that would be nice, but I think this documentation belongs somewhere else, not in the "Getting Started" guide.

**Comment by rofe** [comment]
@tripodsan: @dplaton added all the relevant ones, ok to merge?

**Comment by tripodsan** [comment]
now that we have support for all env variables, why not document all of them?
see https://github.com/adobe/helix-cli/blob/v0.14.12/test/fixtures/all.env

**Comment by rofe** [comment]
I'm sort of with @dplaton on this one - I think the getting started guide should stay concise. I also created another issue to document _all_ env variables a while back: https://github.com/adobe/helix-cli/issues/698.

Alternatively, we could just document them all in 1 place, and then link to that doc from getting started.

File: getting-started.md [hunk 1]
```diff
@@ -163,3 +163,19 @@ Run:
 ```bash
 curl -v -H "X-Debug: true" https://<your_domain>/
 ```
+
+### (Optional) Use environment variables
+
+The authentication parameters used by the `hlx deploy` and `hlx publish` tasks can also be specified as environment variables. You can put this variables into an `.env` file which will be automatically picked up the the client.
```
Comment (tripodsan) [inline getting-started.md]: I would write:

All command line arguments can also be provided as _environment variables_, by prefixing them with `HLX_`. For example the `--log-level` can be set via the `HLX_LOG_LEVEL` variable; the `--wsk-namespace` via `HLX_WSK_NAMESPACE` etc. You can put those variables into an `.env` file which will be automatically picked up by the cli.

File: getting-started.md [hunk 2]
```diff
@@ -163,3 +163,19 @@ Run:
...
+- `HLX_WSK_NAMESPACE`
+- `HLX_WSK_HOST`
+- `HLX_WSK_AUTH` 
+- `HLX_CIRCLECI_AUTH` 
+- `HLX_FASTLY_NAMESPACE`
+- `HLX_FASTLY_AUTH`
+- `HLX_LOGGLY_AUTH`
+- `HLX_LOGGLY_HOST`
+
+
```
Comment (tripodsan) [inline getting-started.md]: Useful variables during development with `hlx up` are also
- `HLX_LOG_LEVEL`
- `HLX_HOST`
- `HLX_LOCAL_REPO`
- `HLX_OPEN`

**Pull request merged**